tests/PHPStan/Rules/Variables/DefinedVariableRuleTest.php

```diff
@@ -1169,4 +1169,19 @@ public function testBug8719(): void
 		$this->analyse([__DIR__ . '/data/bug-8719.php'], []);
 	}
 
+	public function testBug13353(): void
+	{
+		$this->cliArgumentsVariablesRegistered = true;
+		$this->polluteScopeWithLoopInitialAssignments = true;
+		$this->checkMaybeUndefinedVariables = true;
+		$this->polluteScopeWithAlwaysIterableForeach = true;
+
+		$this->analyse([__DIR__ . '/data/bug-13353.php'], [
+			[
+				'Variable $bar might not be defined.',
+				9,
```

Comment:
```suggestion
				11,
```
with the namespace it changed the line ;)

Reply:
Really thank you for your feedbacks !